indexing-service/src/main/java/org/apache/druid/indexing/overlord/http/OverlordRedirectInfo.java

```diff
@@ -37,6 +39,16 @@ public class OverlordRedirectInfo implements RedirectInfo
       "/druid/indexer/v1/isLeader"
   );
 
+  private static final Set<String> LOCAL_GET_PATHS = ImmutableSet.of(
+      "/druid/indexer/v1/task/*/log",
+      "/druid/indexer/v1/task/*/reports",
+      "/druid/indexer/v1/task/*/segments",
```

Comment:
`/druid/indexer/v1/task/*/segments` is backed by a metadata store query, so I suspect scaling this one out across Overlords won't be super useful. (Unless the concern is that the leader might be busy doing _other_ stuff: then it'd be useful.)

Reply:
Yes, primarily the PR is for the situation where Overlord is super busy: To ensure that other features can still work by delegating to followers.